DNA-RNAclasses.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+
+#Dna class. Inherited from str slass.
+class Dna(str):
+    def __init__(self, sequence):
+        DNAcharlist = ('A', 'C', 'G', 'T', 'S', 'W', 'R', 'Y', 'K', 'M', 'B', 'D', 'H', 'V', 'N')
+        badseq = False
+        trueseq = ''
+        for char in sequence:
+            if char.upper() in DNAcharlist:
+                trueseq += char
+            else:
+                badseq = True
+                continue
+        self.sequence = trueseq
+        if badseq:
+            print ('While initializing Dna object non-DNA characters were omitted')
+        if len (sequence) == 0:
+            print ('Empty DNA sequence was created. Using gc() function will return None')
+            
+    def __repr__(self):
+        return (self.sequence)
+    
+    def __str__(self):
+        return (self.sequence)
+            
+#transcribe function returns transcription in coordinates [transcrstart : transcrend] as an Rna class object
+#whole sequence is transcribed by default
+    def transcribe(self, transcr_start = 1, transcr_end = None): 
```

Comment:
This method isn't working - I suppose you've got a mistyping.
 NameError: name 'transcrstart' is not defined

Reply:
Sorry, my bad. Fixed that.